core/src/main/resources/schema/app/20200428235034_DISTX-421_add_scalingmode_to_instancegroup.sql

```diff
@@ -0,0 +1,12 @@
+-- // DISTX-421 Add scalingmode column to instancegroup table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE instancegroup ADD IF NOT EXISTS scalingmode VARCHAR(15);
```

Comment:
`VARCHAR(15)` this seems a little bit restrictive. I think we usually use 255 for enums

Reply:
Thanks, done!